client-java/controller-api/src/main/java/org/evomaster/client/java/controller/api/dto/problem/rpc/RPCActionDto.java

```diff
@@ -113,6 +114,11 @@ public class RPCActionDto {
      */
     public boolean doGenerateTestScript;
 
+    /**
+     * the output format
+     */
+    public SutInfoDto.OutputFormat outputFormat = SutInfoDto.OutputFormat.JAVA_JUNIT_5;
```

Comment:
why this default value of Java JUnit 5?

Reply:
this info will sync based on what the user specified in EMConfig or driver.
this doc has been updated now.

Reply:
sorry. i don't understand this. why not leaving it null? i find bit confused about having DTOs with default non-null values.